docs/whats-new/dotnet-aspire-9.1.md

```diff
@@ -1,18 +1,117 @@
+
+### Localization overrides
+
+The dashboard defaults to the language that your browser is in. In this release, we added the ability to override that and change the language the dashboard is in separately from the browser language.
+![A screenshot of the aspire dashboard showing the new flyout menu to change language](./media/dashboard-language.png)
+
+### Filtering
+
+You can now filter what you see in the Resource view by type, state, and health status.
+
+### More resource details
+
+When you click on a resource in the dashboard, more data now appears in the details pane, including references, back references, and volumes with their mount types.
+
+![A screenshot of the aspire dashboard with references and back references showing](./media/dashboard-resourcedetails.png)
+
+### CORS support for custom local domains
+
+You can now set the `DOTNET_DASHBOARD_CORS_ALLOWED_ORIGINS` environment variable to allow the dashboard to receive telemetry from other browser apps, such as if you have resources running on custom localhost domains.
+
+### Flexibility with console logs
+
+The console log page has two new options. You're now able to download your logs so you can view them in your own diagnostics tools. Plus, you can turn timestamps on or off to reduce visual clutter when needed.
+
+![A screenshot of the console logs page with the download button, turn off timestamps button, and logs that do not show timestamps](./media/consolelogs-download.png)
+
+### Various UX improvements
+
+There are a bunch of new features in 9.1 that streamline and polish popular tasks, including:
+
+- Resource commands, such as Start/Stop buttons, now also appears on the console logs page
+- Single click to open in text visualizer
+- URLs are automatically clickable, and there are no more commas separating endpoints
+
+...and, most importantly, your scroll position now resets when you click between different resources 🥳
+
+There are many more small tweaks that add up to a smoother dashboard experience - for some others, you can check out [James Newton-King](https://bsky.app/profile/james.newtonking.com) on Bluesky, who has been posting new dashboard features daily for the last few weeks!
+
+## Local dev
+
+### Start resources on demand
+
+You can now tell resources not to start with the rest of your app by using `WithExplicitStart()` on the resource in your app host. Then, you can start it whenever you're ready from inside the dashboard.
+
+### Better Docker integration
+
+We added the ability to `PublishAsDockerfile()` for any project or container resource, and you can overload existing Dockerfiles to customize them further in the app host.
+
+### Cleaning up Docker networks
+
+We fixed [an annoying bug](https://github.com/dotnet/aspire/issues/6504) in 9.1 where Docker networks created by Aspire would stay alive even after you stopped running the app. We now clean up after ourselves!
+
+## Integrations
```

Comment:
@captainsafia We're missing an entire section on existing resources.

@IEvangelist we're going to have to update the azure integration docs as well.

Reply:
The docs issue filed over at https://github.com/dotnet/docs-aspire/issues/2549 can be used to source material for this.